core/templates/dev/head/pages/topic_editor/subtopics_editor/SubtopicsListTabDirective.js

```diff
@@ -34,90 +34,155 @@ oppia.directive('subtopicsListTab', [
             UndoRedoService, SubtopicPageObjectFactory,
             EVENT_TOPIC_REINITIALIZED, EVENT_TOPIC_INITIALIZED,
             EVENT_SUBTOPIC_PAGE_LOADED) {
-          // The subtopic preview/editor would be hidden until a subtopic is
-          // clicked.
-          $scope.subtopicDisplayed = false;
           var _initEditor = function() {
             $scope.topic = TopicEditorStateService.getTopic();
             $scope.subtopics = $scope.topic.getSubtopics();
             $scope.subtopicEditorIsShown = false;
+            $scope.uncategorizedSkillIds =
```

Comment:
As above -- should have ID + description be one "thing", conceptually.

Reply:
Done.